cmd/fmt.go

```diff
@@ -25,7 +25,7 @@ var fmtParams = struct {
 }{}
 
 var formatCommand = &cobra.Command{
-	Use:   "fmt",
+	Use:   "fmt <path> [path [...]]",
```

Comment:
nit: instead of `<path> [path [...]]` can we do `[path [...]]` since the path is optional. On some of the other commands like `opa test` we have it show up as
```
Usage:
  opa test <path> [path [...]] [flags]
```
And it requires that `<path>` otherwise you get an error. So for consistency lets keep the `path` documented as optional.

I think adding in a sentence like `If no path is provided the rego content will be read from stdin` might also help clear things up a bit more.

Reply:
ah good suggestion! Sorry I missed this in the code -- i'll add both!